inf-clojure.el

```diff
@@ -770,6 +806,64 @@ Return the number of nested sexp the point was over or after."
   (setq-local eldoc-documentation-function #'inf-clojure-eldoc)
   (apply #'eldoc-add-command inf-clojure-extra-eldoc-commands))
 
-(provide 'inf-clojure)
+;;;; Lumo
+;;;; ====
+
+(defgroup lumo nil
+  "Run an external Lumo process (REPL) in an Emacs buffer."
+  :group 'inf-clojure)
 
+(defcustom inf-clojure-lumo-command "lumo"
+  "The command used to launch lumo."
+  :type 'string
+  :group 'lumo)
+
+(defcustom inf-clojure-lumo-args "-d"
+  "The command arguments used to launch lumo."
+  :type 'string
+  :group 'lumo)
+
+;; AR - TODO Alternatively you can specify a command string that will be called,
+;; which should return a string.
+(defcustom inf-clojure-lumo-classpath-generator "cp"
+  "The file used to create the classpath string.
+The classpath string has to be a \":\" separated list of dir and
+files."
+  :type 'string
+  :group 'lumo)
+
+;; AR - not used but left here because it is a possible sanity check
+(defun inf-clojure--lumo-mode-p ()
+  "Return true if the lumo is the target REPL."
+  (comint-send-string (inf-clojure-proc) "(js/global.hasOwnProperty \"$$LUMO_GLOBALS\")"))
+
+(defun inf-clojure--lumo-program ()
```

Comment:
I think this name is a bit confusing. This doesn't return the lumo program, but some command complete with parameters.